docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -1,25 +1,430 @@
-Please remind, that Bareos Director and Bareos Storage Daemon must always have the same version. The version of the File Daemon may differ, see chapter about :ref:`backward compatibility <backward-compatibility>`.
+
+.. note::
+
+   Please before processing, apply steps in :ref:`section-UpdateUpgradePreamble`
+
+
+In this section, we explain how to update your Bareos major version to the latest minor or bugfix release.
+
+For upgrading to a new major version see :ref:`Update Bareos to a new major release <section-UpgradeMajor>`.
+
+.. note::
+
+   You can install directly the latest Major,Minor,Bugfix release available.
+   So updating from 21.0.0 directly to 21.1.5 is not a problem.
+
+
+
+Example how to update from 21.0.0 to 21.1.5.
+
+.. index::
+   single: update minor bugfix; RHEL
+   single: update minor bugfix; CentOS
+   single: update minor bugfix; Fedora
+   single: update minor bugfix; EL
+
+.. code-block:: shell-session
+   :caption: Shell example command to update Bareos on on EL 8
+
+   root@host:~# dnf upgrade --repo=bareos --refresh
+      Backup Archiving Recovery Open Sourced (EL_8)                                                              1.5 kB/s | 833  B     00:00
+      Dependencies resolved.
+      ====================================================================================
+      Package                         Architecture     Version         Repository   Size
+      ====================================================================================
+      Upgrading:
+      bareos                          x86_64           21.1.5-3.el8    bareos      7.4 k
+      bareos-bconsole                 x86_64           21.1.5-3.el8    bareos       37 k
+      bareos-client                   x86_64           21.1.5-3.el8    bareos      7.5 k
+      bareos-common                   x86_64           21.1.5-3.el8    bareos      764 k
+      bareos-database-common          x86_64           21.1.5-3.el8    bareos       87 k
+      bareos-database-postgresql      x86_64           21.1.5-3.el8    bareos       42 k
+      bareos-database-tools           x86_64           21.1.5-3.el8    bareos      107 k
+      bareos-director                 x86_64           21.1.5-3.el8    bareos      425 k
+      bareos-filedaemon               x86_64           21.1.5-3.el8    bareos      120 k
+      bareos-storage                  x86_64           21.1.5-3.el8    bareos       97 k
+      bareos-tools                    x86_64           21.1.5-3.el8    bareos       52 k
+
+      Transaction Summary
+      ====================================================================================
+      Upgrade  11 Packages
+
+      Total download size: 1.7 M
+      Is this ok [y/N]: y
+
+
+.. index::
+   single: update minor bugfix; SLE
+   single: update minor bugfix; openSUSE
+
+.. code-block:: shell-session
+   :caption: Shell example command to update Bareos on SLES / openSUSE
+
+   root@host:~# zypper refresh --force bareos
```

Comment:
Why `--force`? Force always leaves the impression, that you do something you shouldn'd do.

Reply:
because if the project publish something recently (between the lap of time the cache is valid) the metadata will not be in sync with the repository content. Maybe because I'm use to Tumbleweed and get often hit by this kind of things.